correlator_pcaspy.py

```diff
@@ -99,25 +90,26 @@ def __init__(self, host: str, pv_prefix: str, firmware_revision: str, filepath:
             Records.OVERLOADINTERVAL.value: 400,
             Records.REPETITIONS.value: 2,
             Records.CURRENT_REPETITION.value: 0,
-            Records.CONNECTED.value: self.device.isConnected(),
+            Records.CONNECTED.value: self.driver.is_connected,
             Records.RUNNING.value: False,
-            Records.SCATTERING_ANGLE.value: 2.2,
-            Records.SAMPLE_TEMP.value: 300,
-            Records.SOLVENT_VISCOSITY.value: 1100,
-            Records.SOLVENT_REFRACTIVE_INDEX.value: 1.1,
-            Records.LASER_WAVELENGTH.value: 440,
+            Records.SCATTERING_ANGLE.value: 110,
+            Records.SAMPLE_TEMP.value: 298,
+            Records.SOLVENT_VISCOSITY.value: 1,
+            Records.SOLVENT_REFRACTIVE_INDEX.value: 1.33,
+            Records.LASER_WAVELENGTH.value: 642,
```

Comment:
These default values were wrong, and confused the user when they used it last. They're the acceptance criteria from another ticket, but I implemented it here because it's such a trivial change: https://github.com/ISISComputingGroup/IBEX/issues/5303